sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_aws_sdk.ipynb

```diff
@@ -0,0 +1,889 @@
+   "metadata": {},
+   "source": [
+    "### Explaining Predictions\n",
+    "There are expanding business needs and legislative regulations that require explanations of _why_ a model made the decision it did. SageMaker Clarify uses SHAP to explain the contribution that each input feature makes to the final decision."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configure a SageMaker Clarify Processing Job Container's Input and Output Parameters "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "explainability_analysis_config_path = \"s3://{}/{}/explainability_analysis_config.json\".format(\n",
+    "    bucket, prefix\n",
+    ")\n",
+    "analysis_result_path = \"s3://{}/{}/output\".format(bucket, prefix)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configure Analysis Config\n",
+    "For our example use case we will be using the following analysis config:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!echo\n",
+    "!cat explainability_analysis_config.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`explainability_analysis_config.json` here contains configuration values for detecting bias using a clarify container:\n",
+    "* `dataset_type` specify the format of your dataset, for thsi example as we are using csv dataset this will be `text/csv`\n",
+    "* `headers` is the list of column names in the dataset\n",
+    "* `label` specified the target attribute for the model to be used for bias metrics. Specified as a column name, or as index if the dataset format is CSV\n",
+    "* `method` is the list of methods and their parameters for the analyses and reports. If any section is omitted, then it is not computed.\n",
+    "  * **shap:** Kernel SHAP algorithm requires a baseline (also known as background dataset). If not provided, a baseline is calculated automatically by SageMaker Clarify using K-means or K-prototypes in the input dataset. Baseline dataset type shall be the same as `dataset_type`, and baseline samples shall only include features. By definition, `baseline` should either be a S3 URI to the baseline dataset file, or an in-place list of samples. In this case we chose the latter, and put the first sample of the test dataset to the list.\n",
+    "\n",
+    "* `instance_type` and `instance_count` specify your preferred instance type and instance count used to run your model on during SageMaker Clarify's processing. The testing dataset is small so a single standard instance is good enough to run this example. If your have a large complex dataset, you may want to use a better instance type to speed up, or add more instances to enable Spark parallelization.\n",
```

Comment:
Major comment:
* The configurations are under the "predictor" section.
* "model_name"?
* `initial_instance_count` instead of `instance_count`. And remove "If your have a large complex dataset, you may want to use a better instance type to speed up, or add more instances to enable Spark parallelization." which is a comment for processing instance but here we are talking about endpoint instance.